float32 lattitude
float32 longitude
float32 altitude
float32 distance
